src/mongo/db/server_options.h

```diff
@@ -66,6 +66,7 @@ struct ServerGlobalParams {
     int defaultProfile = 0;                // --profile
     int slowMS = 100;                      // --time in ms that is "slow"
     int rateLimit = 1;                     // --rate limit in the range 1-RATE_LIMIT_MAX represents a  1/N probability that a query will be profiled
+    long long collScanLimit = -1;  // --COLLSCANs with this number or more scanned docs will be logged (-1 means no COLLSCANs will be logged)
```

Comment:
Made 0 the default value meaning no special handling for COLLSCANs. Changed range checks so that now no negative values are allowed.
Left type `long long` to simplify comparisons with `docsExamined`.